server/app/controllers/GitHubServer.scala

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.Uri.{Path, Query}
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.{Accept, Authorization, GenericHttpCredentials}
+import com.typesafe.config.ConfigFactory
+
+import scala.concurrent.Future
+
+
+trait GitHubServer {
```

Comment:
Do we really need to implement everything from scratch? Why don't we use https://github.com/scribejava/scribejava ?